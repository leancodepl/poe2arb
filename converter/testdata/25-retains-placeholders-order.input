[
    {
        "term": "home_SiteContact",
        "definition": "{fullName}, Phone: {phoneNumber}, Email: {emailAddress}",
        "context": "",
        "term_plural": "",
        "reference": "",
        "comment": ""
    }
]