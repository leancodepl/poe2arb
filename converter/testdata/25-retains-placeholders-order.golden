{
    "@@locale": "en",
    "home_SiteContact": "{fullName}, Phone: {phoneNumber}, Email: {emailAddress}",
    "@home_SiteContact": {
        "placeholders": {
            "fullName": {
                "type": "Object"
            },
            "phoneNumber": {
                "type": "Object"
            },
            "emailAddress": {
                "type": "Object"
            }
        }
    }
}
